ark/settings.py

```diff
@@ -1,7 +1,7 @@
 from decouple import config
 
 
-__version__ = '0.1.2'
+__version__ = '0.1.1'
```

Comment:
as state above

## pt

Realizar o comando `bumpversion patch` como descrito nas orientações de contribuição

## en

Make the `bumpversion patch` command as described on the contribution guidelines